mnav_points: [3, 0, 2]
